ruby_review_week_four.rb

```diff
@@ -1,37 +1,95 @@
 
-# 11. 
+# 11.
 
-# a. Create one fish object and store it in a variable, and 
+# a. Create one fish object and store it in a variable, and
+
+fish1 = Fish.new
+fish1.name = 'fred'
+fish1.species = 'stuff'
 
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
-# 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key. 
+# 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key.
 
-# 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
+student = {
+  name: 'Sponge Bob',
+  age: 99
+}
 
-# 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
+# 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
 
-# 15. Write a method that takes one argument, a number, and that returns true if the number is divisible by three, and false otherwise. 
+def sum(num1, num2, num3)
+  num1 + num2 + num3
+end
 
-# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
\ No newline at end of file
+# 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
+def true_string(arr, str)
+  arr.each do |i|
+    return true if i == str
+  end
+  false
+end
+
+# 15. Write a method that takes one argument, a number, and that returns true if the number is divisible by three, and false otherwise.
+def div_three(num)
+  num % 3 == 0
+end
+# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
+
+def double_array(arr)
+  arr_div_three = []
+  arr_not_div_three = []
+  arr.each do |number|
+    if div_three(number)
+      arr_div_three.push(number)
+    else
+      arr_not_div_three.push(number)
+    end
+  end
+  result = [arr_div_three, arr_not_div_three]
```

Comment:
this is correct, consider just returning the array rather than making a variables